cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -71,10 +71,12 @@ kubectl apply -f examples/cluster-autoscaler-run-on-master.yaml
 
 ### Auto-Discovery Setup
 
-To run a cluster-autoscaler which auto-discovers ASGs with nodes use the `--node-group-auto-discovery` flag and tag the ASGs with _key_ `k8s.io/cluster-autoscaler/enabled` with value `yes` and _key_ `kubernetes.io/cluster/<YOUR CLUSTER NAME>`with value `true`.
```

Comment:
The information on how to tag the ASGs seems to be lost in this rewrite, is this intentional?

Reply:
That's what I'm updating. The app looks for the existence of the tag(s) in the `--node-group-auto-discovery` flag but doesn't look at their values. 

My autoscaling group has these tags

```
$> aws autoscaling describe-tags --filters Name=auto-scaling-group,Values=k8s-worker.touge.k8s.tubemogul.info | jq '.Tags[] | select(.PropagateAtLaunch == false) | .'                              
{
  "ResourceType": "auto-scaling-group",
  "ResourceId": "k8s-worker.touge.k8s.tubemogul.info",
  "PropagateAtLaunch": false,
  "Value": "",
  "Key": "k8s.io/cluster-autoscaler/enabled"
}
{
  "ResourceType": "auto-scaling-group",
  "ResourceId": "k8s-worker.touge.k8s.tubemogul.info",
  "PropagateAtLaunch": false,
  "Value": "",
  "Key": "k8s.io/cluster-autoscaler/touge.k8s.tubemogul.info"
}
```

and the autoscaler with

```
$> kubectl describe pod -n kube-system cluster-autoscaler-84cf96ffdd-8kpdx | grep "node-group-auto-discovery"
      --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,k8s.io/cluster-autoscaler/touge.k8s.tubemogul.info
```

detects the ASG properly and scales appropriately.

```
I0601 17:17:11.968302       1 aws_manager.go:185] Autodiscovered ASG k8s-worker.touge.k8s.tubemogul.info using tags [k8s.io/cluster-autoscaler/enabled k8s.io/cluster-autoscaler/touge.k8s.tubemogul.info]
I0601 17:17:11.968311       1 auto_scaling_groups.go:153] Regenerating instance to ASG map for ASGs: [k8s-worker.touge.k8s.tubemogul.info]
I0601 17:17:12.046559       1 aws_manager.go:241] Refreshed ASG list, next refresh after 2018-06-01 17:18:12.046555345 +0000 UTC
```

When I change the values of the tags

```
$> aws autoscaling describe-tags --filters Name=auto-scaling-group,Values=k8s-worker.touge.k8s.tubemogul.info | jq '.Tags[] | select(.PropagateAtLaunch == false) | .'                              
{
  "ResourceType": "auto-scaling-group",
  "ResourceId": "k8s-worker.touge.k8s.tubemogul.info",
  "PropagateAtLaunch": false,
  "Value": "foo",
  "Key": "k8s.io/cluster-autoscaler/enabled"
}
{
  "ResourceType": "auto-scaling-group",
  "ResourceId": "k8s-worker.touge.k8s.tubemogul.info",
  "PropagateAtLaunch": false,
  "Value": "foo",
  "Key": "k8s.io/cluster-autoscaler/touge.k8s.tubemogul.info"
}
```

the autoscaler still detects the ASG.

```
I0601 17:22:37.472898       1 aws_manager.go:185] Autodiscovered ASG k8s-worker.touge.k8s.tubemogul.info using tags [k8s.io/cluster-autoscaler/enabled k8s.io/cluster-autoscaler/touge.k8s.tubemogul.info]
I0601 17:22:37.472909       1 auto_scaling_groups.go:153] Regenerating instance to ASG map for ASGs: [k8s-worker.touge.k8s.tubemogul.info]
I0601 17:22:37.558116       1 aws_manager.go:241] Refreshed ASG list, next refresh after 2018-06-01 17:23:37.558108991 +0000 UTC
```